mtc.go

```diff
@@ -670,11 +623,13 @@ func (p *CAParams) newTreeHeads(prevHeads, root []byte) ([]byte, error) {
 	return append(prevHeads[HashLen:len(prevHeads)], root...), nil
 }
 
-func (batch *Batch) Anchor() *MerkleTreeTrustAnchor {
-	return &MerkleTreeTrustAnchor{
-		issuerId:    batch.CA.IssuerId,
-		batchNumber: batch.Number,
+func (batch *Batch) Anchor() *TrustAnchorIdentifier {
+	tai := TrustAnchorIdentifier{}
+	err := tai.UnmarshalText([]byte(fmt.Sprintf("%s.%d", batch.CA.IssuerOID, batch.Number)))
```

Comment:
I guess it works. Perhaps neater to add a function on `TrustAnchorIdentifer` which does this more directly.